### PR TITLE
413 / withdraw loading

### LINKED ISSUE #413: As a user I need the platform to indicate that a tx is going through when i'm withdrawing a property in order to indicate the waiting time of the tx
After confirming the tx when withdrawing a property, there is a technical time while the result is shown under "My Properties" (both All and Lent tabs). In this case, so it doesn't seem like the system is broken, we need to catch the event and display this "loading" state of the card.

There is a [design for the loading state of a property card](https://www.figma.com/file/gMFxv9DjmUkHbZQSe4lu5E/EnterDAO-Projects?node-id=5165%3A15579).

This is same as #210 and #386but for renting rather than listing.

AC: Given the user is **withdrawing** a property when then **withdraw** is going through then the loading state should be shown in the All and Lent tabs in the My Properties page until the tx is completed.

### PR DESCRIPTION
**Pull request by mishaadzh**
**Detailed description**:
display loading when tx is going

**Which issue(s) this PR fixes**:
Fixes #413 #386 

**Special notes for your reviewer**:

**Checklist**
- [x] 413
- [x] 386

### CODE REVIEW

**Comment by failfmi** [comment]
This does not include #386?

File: src/modules/land-works/views/single-land-view/index.tsx [hunk 1]
```diff
@@ -164,6 +164,7 @@ const SingleLandView: React.FC = () => {
 
     try {
       await landWorksContract?.delist(asset.id, () => {
+        localStorage.setItem('WITHDRAW_IN_PROGRESS', asset.metaverseAssetId);
```
Comment (failfmi) [inline src/modules/land-works/views/single-land-view/index.tsx:167]: There's a case when you will not withdraw it when you delist it.

**Review by failfmi — CHANGES_REQUESTED** [review]

**Comment by mishaadzh** [comment]
> This does not include #386?

I still need to test and I will combine these two tasks

File: src/modules/land-works/views/single-land-view/index.tsx [hunk 2]
```diff
@@ -168,6 +168,7 @@ const SingleLandView: React.FC = () => {
...
```
Comment (failfmi) [inline src/modules/land-works/views/single-land-view/index.tsx:171]: There is one more function that withdraws -> `landWorksContract?.withdraw()`

**Review by failfmi — APPROVED** [review]

File: src/modules/land-works/views/single-land-view/index.tsx (hunk 2 above)
Comment (failfmi) [reply src/modules/land-works/views/single-land-view/index.tsx:171]: if the delisting is direct, only then the withdraw should be in progress, if not -> do not set anything in local storage.

File: src/modules/land-works/views/single-land-view/index.tsx [hunk 3]
```diff
@@ -130,6 +130,7 @@ const SingleLandView: React.FC = () => {
     try {
       await landWorksContract?.withdraw(asset.id, () => {
         setWithdrawButtonDisabled(true);
+        localStorage.setItem('WITHDRAW_IN_PROGRESS', asset.metaverseAssetId);
```
Comment (failfmi) [inline src/modules/land-works/views/single-land-view/index.tsx:133]: I guess we need to add removal in catch block similar to `delist`?

**Review by failfmi — APPROVED** [review]

File: src/modules/land-works/views/my-properties-view/index.tsx [hunk 4]
```diff
@@ -158,6 +161,27 @@ const MyPropertiesView: FC = () => {
...
+  };
+
+  const displayExistLandLoader = () => {
+    return (
+      isExistingLandInProgress(lands, loading, 'WITHDRAW_IN_PROGRESS') ||
+      isExistingLandInProgress(lands, loading, 'EXIST_RENT_IN_PROGRESS')
+    );
+  };
+
+  const newProtertyTitle = () => {
```
Comment (failfmi) [inline src/modules/land-works/views/my-properties-view/index.tsx:178]: one small typo here

File: src/modules/land-works/views/my-properties-view/index.tsx [hunk 5]
```diff
@@ -158,6 +161,27 @@ const MyPropertiesView: FC = () => {
...
+    return localStorage.getItem('LISTING_IN_PROGRESS') ? 'Listing' : 'Renting';
+  };
+  const existProtertyTitle = () => {
```
Comment (failfmi) [inline src/modules/land-works/views/my-properties-view/index.tsx:181]: one small typo here

**Review by failfmi — CHANGES_REQUESTED** [review]

**Review by failfmi — APPROVED** [review]

**Pull request merged**